projects/ngrx.io/tsconfig.json

```diff
@@ -3,9 +3,7 @@
   "compilerOptions": {
     "strict": true,
     "noImplicitAny": false,
-    // disabled because this is on by default in tsc 2.7 breaking our codebase - we need to refactor
     "strictPropertyInitialization": false,
-    // disabled because of https://github.com/angular/angular/issues/22877
```

Comment:
these comments should be also reverted

Reply:
Re-added.
Second comment was removed, because the linked issue is resolved.
I tried to fix the first one as well, but there were more violations than expected 😅

Reply:
Yikes, `skipLibCheck` was still needed. re-added, with comment :)